src/main/kotlin/com/hibob/academy/resource/PetResource.kt

```diff
@@ -41,4 +48,18 @@ class PetResource(private val petService: PetService) {
         return Response.ok(petService.updatePetOwner(id, ownerId, companyId)).build()
     }
 
+    @POST
+    @Path("/{companyId}/ownerId/{ownerId}/petIds")
```

Comment:
I think it would be better to give more descriptive name instead of petIds - maybe adoptPets or something